app/jobs/osym/import_prospective_students_job.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+module Osym
+  class ImportProspectiveStudentsJob < ApplicationJob
+    queue_as :high
+
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/BlockLength
+    def perform
+      file = File.open(Rails.root.join('db', 'static_data', 'prospective_students.csv'))
+      progress_bar = ProgressBar.create(
+        title: 'Prospective Students',
+        total: file.readlines.length,
+        format: '%t %B %c/%C %a'
+      )
+
+      file.read.each_line do |prospective_student|
+        id_number, first_name, last_name, fathers_name, mothers_name, d, m, y, gender, turkish, kktc, foreign,
```

Comment:
"Maintainability" için bu civarın saçını başını taramak lazım biraz. Bloke olmayalım.  Gelecekte bakarı(mz).